checks.d/mesos_master.py

```diff
@@ -190,7 +194,13 @@ def check(self, instance):
         default_timeout = self.init_config.get('default_timeout', 5)
         timeout = float(instance.get('timeout', default_timeout))
 
-        state_metrics = self._check_leadership(url, timeout)
+        no_ssl = _is_affirmative(instance.get('disable_ssl_validation', False))
+        ssl_verify = False if no_ssl else True
+        parsed_url = urlparse(url)
+        if no_ssl and parsed_url.scheme == 'https':
+            self.log.warning('Skipping SSL cert validation for %s based on configuration.' % url)
```

Comment:
I agree with Yann, logging it once (first check run) would make sense, but it would otherwise spam the logs for nothing. 😢  (not saying skipping SSL cert is not a big deal, but I hate it when programs remind every 5min (in this case every 20s) of a decision I made, fully aware of the consequences).